conans/client/cmd/new.py

```diff
@@ -236,13 +236,38 @@ def test(self):
 
 """
 
+def _get_files_from_template_dir(template_dir, name, version, package_name):
+    if not os.path.isfile(os.path.join(template_dir, "conanfile.py")):
+        raise ConanException("Template is missing 'conanfile.py': {}".format(template_dir))
+
+    render_args = {'name': name, 'version': version, 'package_name': package_name}
```

Comment:
```suggestion
    context = {'name': name, 'version': version, 'package_name': package_name}
```

I'd add at least the Conan version to this context.

Reply:
Is it this one?
`from conans import __version__ as client_version`

Reply:
Yes, that one, probably wrapped in a `Version` object:

```python
from conans.client.tools.version import Version

context = {...., 'version': Version(client_version)}
```

Reply:
Fixed.
Used plain str, cause `Version(ver)` rendered as `Object...`

Reply:
Ok, we can change it for a `Version` object if needed, and add a `__str__` to the class. But that can wait 👍